sync/class.jetpack-sync-module-callables.php

```diff
@@ -9,7 +9,7 @@ class Jetpack_Sync_Module_Callables extends Jetpack_Sync_Module {
 	private $callable_whitelist;
 
 	public function name() {
-		return "callables";
+		return 'functions';
```

Comment:
This was so that I didn't have to have 2 different names for the same module, one used in the "full sync status" hash, and another used as an ID internally.